packages/editor/CodeMirror/editorCommands/duplicateLine.ts

```diff
@@ -0,0 +1,34 @@
+import { EditorSelection } from '@codemirror/state';
+import { Command, EditorView } from '@codemirror/view';
+
+const duplicateLine: Command = (editor: EditorView) => {
+	const state = editor.state;
+	const doc = state.doc;
+
+	const transaction = state.changeByRange(range => {
```

Comment:
Currently, this duplicates selected lines multiple times if there are multiple cursors on a line. However, because the CodeMirror 5 editor has the same behavior, I think it's fine to leave this as-is.